assets/css/style.css

```diff
@@ -0,0 +1,1262 @@
+@import url("../fonts/Lato-Light.ttf");
+@import url("../fonts/Cocogoose Pro Semilight-trial.ttf");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
```

Comment:
- [ ] Your fony size is ver small. Kindly apply the right size as in the given design: 
![image](https://user-images.githubusercontent.com/91486791/203617560-fc6a2363-d668-4b92-bc9e-8a8c8c129657.png)
your design: 
![image](https://user-images.githubusercontent.com/91486791/203617617-be7f8375-e49c-4f38-99ef-adfdb5065494.png)